fuse/server.go

```diff
@@ -560,13 +552,13 @@ func (ms *Server) EntryNotify(parent uint64, name string) Status {
 
 // SupportsVersion returns true if the kernel supports the given
 // protocol version or newer.
-func (in *InitIn) SupportsVersion(maj, min uint32) bool {
+func (in InitIn) SupportsVersion(maj, min uint32) bool {
```

Comment:
I didn't take that approach because I figured the goal was to give the user something that wouldn't mutate the original.